modules/gestor-usuarios/usuarios.routes.ts

```diff
@@ -44,7 +44,7 @@ class UsuariosResource extends ResourceBase {
             const user = await getUserInfo(documento);
             return res.json(user);
         } catch (err) {
-            throw new ResourceNotFound();
+            throw new Error('No se pudo obtener información del usuario en One Login');
```

Comment:
Es un detalle, pero la API devuelve 500 y eso se interpreta como error del sistema. Visible desde APM por ejemplo.

```suggestion
            throw new CustomError('No se pudo obtener información del usuario en One Login', 404);
```

CustomError pertenece a @andes/core.